addons/im_livechat/tests/test_message.py

```diff
@@ -89,7 +89,7 @@ def test_message_format(self):
             'message_type': 'notification',
             'reactions': [],
             'model': 'discuss.channel',
-            'needaction_partner_ids': [],
+            'needaction': False,
```

Comment:
Maybe it would be time to rename it to something closer than what it actually is? `is_in_inbox` maybe?

Reply:
I agree, but it's a lot to rename, maybe in a follow-up PR?

Reply:
Just the needaction field for now, you're already renaming it/introducing it in this PR.